### PR TITLE
Fixing CustomEvent polyfill for IE11

### PR DESCRIPTION
**Pull request by camelburrito**
Fixes : https://github.com/ampproject/amphtml/issues/8916

### CODE REVIEW

File: src/event-helper.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ const LOAD_FAILURE_PREFIX = 'Failed to load:';
  * @return {!Event}
  */
 export function createCustomEvent(win, type, detail) {
-  if (win.CustomEvent) {
+  // win.CustomEvent is a function on Edge, Chrome, FF, Safari but
+  // is an object on IE 11.
+  if (typeof(win.CustomEvent) == 'function') {
```
Comment (jridgewell) [inline src/event-helper.js]: Nit: no parenthesis, `typeof` is a unary operator not a function.

File: test/functional/test-event-helper.js [hunk 2]
```diff
@@ -218,4 +218,25 @@ describe('EventHelper', () => {
...
   });
+
+  it('should create the correct custom event for IE11', () => {
+    const native = createCustomEvent(window, 'foo', {bar: 123});
+    expect(native.type).to.equal('foo');
+    expect(native.detail).to.deep.equal({bar: 123});
+
+    const initCustomEventSpy = sandbox.spy();
+    const win = {};
+    win.customEvent = {};
```
Comment (dvoytenko) [inline test/functional/test-event-helper.js]: `CustomEvent`, i.e. capitalized.

Comment (camelburrito) [inline test/functional/test-event-helper.js]: Done

File: src/event-helper.js (hunk 1 above)
Comment (camelburrito) [inline src/event-helper.js]: Done

**Pull request merged**